angular/src/assets/i18n/es.json

```diff
@@ -0,0 +1,179 @@
+{
+    "buttons": {
+        "cancel": "Cancelar",
+        "close": "Cerrar",
+        "send": "Enviar",
+        "login": "Acceder",
+        "register": "Registrar",
+        "bookTable": "Reservar una mesa",
+        "inviteFriends": "Invitar amigos",
+        "viewMenu": "Ver el menú",
+        "addToOrder": "Añadir a la orden",
+        "clearFilters": "Limpiar filtros",
+        "applyFilters": "Aplicar filtros",
+        "removeComment": "Eliminar comentario",
+        "addComment": "Añadir comentario"
+    },
+    "table": {
+        "rowsPage": "Filas por página",
+        "of": "de",
+        "noResults": "Sin resultados que mostrar"
+    },
+    "alerts": {
+        "accessError": "Acceso prohibido, por favor identifíquese primero",
+        "genericError": "Ha ocurrido un error, por favor vuelva a intentarlo más tarde",
+        "urlError": "Url no encontrada, por favor vuelva a intentarlo",
+        "email confirmations": {
+            "invitationAccept": "Invitación aceptada satisfactoriamente",
+            "invitationReject": "Invitación cancelada satisfactoriamente",
+            "bookingCancel": "Reserva cancelada satisfactoriamente",
+            "orderCancel": "Comanda cancelada satisfactoriamente"
+        },
+        "orderID": "Para hacer una comanda es necesario tener una ID de reserva. Introduce el ID de la mesa que has reservado o bien resérvala ahora mismo.",
+        "passwordError": "La contraseña no coincide",
+        "authAlerts": {
+            "loginSuccess": "Acceso correcto",
+            "registerSuccess": "Registro satisfactorio",
+            "registerFail": "El registro ha fallado, nombre de usuario ya en uso",
+            "logoutSuccess": "Sesión cerrada con éxito, vuelve pronto!"
```

Comment:
Shouldn't it be "Sesión cerrada con éxito, ¡vuelve pronto!" ? It seems there is a lacking `¡` .

Reply:
right, I have to commit again with some enhancements, it will be corrected in there. Thank you!